pyoos/parsers/nerrs.py

```diff
@@ -116,7 +119,7 @@ def get_station(feature):
             s = Station()
             s.uid = feature['Station_Code']
             s.name = feature['Station_Name']
-            s.location = sPoint(float(feature['Longitude']), float(feature['Latitude']), 0)
+            s.location = sPoint(feature['Longitude'], feature['Latitude'], 0)
```

Comment:
@emiliom I don't know the first thing about NERRS, nor I can test it, but isn't it safer to cast the position to `float` here?